src/test/java/bio/terra/service/resourcemanagement/google/BucketResourceTest.java

```diff
@@ -158,12 +161,11 @@ public void createAndDeleteBucketTest() throws Exception {
   // create buckets in different regions and see if they're actually created there.
   public void createBucketsInDifferentRegionsTest() throws Exception {
     for (GoogleRegion region : List.of(GoogleRegion.US_CENTRAL1, GoogleRegion.US_EAST1)) {
-      String bucketName = "testbucket_bucketregionstest_" + region;
+      String bucketName = createBucketName(region.toString());
```

Comment:
Did you intend to remove the test name prefix from the bucket here?
```suggestion
      String bucketName = createBucketName("bucketregionstest_" + region);
```

Reply:
Yes. GCS requires that bucket names be < 64 characters and we were over otherwise.